examples/simple/lib/simple.ex

```diff
@@ -21,10 +21,10 @@ defmodule Weather do
   @primary_key {:id, :binary_id, autogenerate: true}
 
   schema "weather" do
-    field :city, :string
-    field :temp_lo, :integer
-    field :temp_hi, :integer
-    field :prcp, :float, default: 0.0
+    field(:city, :string)
```

Comment:
here we may want to allow no parenthesis CF: https://github.com/elixir-ecto/ecto/blob/master/.formatter.exs